lib/ansible/module_utils/sops.py

```diff
@@ -0,0 +1,60 @@
+# Copyright (c), Edoardo Tenani <e.tenani@arduino.cc>, 2018-2020
+# Simplified BSD License (see licenses/simplified_bsd.txt or https://opensource.org/licenses/BSD-2-Clause)
+
+from __future__ import absolute_import, division, print_function
+
+from ansible.errors import AnsibleError
```

Comment:
Hmm, this is in fact a problem. Everything in `module_utils` can only use stuff from `module_utils`, but not from the rest of Ansible. That's somewhat unfortunate.

How about having a function `createSopsErrorMessage()` instead which is used by `SopsError` in the plugin to create the message? Then the heavy part (`sops_error_codes` and the logic to use it) is in `module_utils`, but `module_utils/sops.py` doesn't need to know about `AnsibleError`.